cpu/sam0_common/periph/rtc.c

```diff
@@ -65,8 +65,8 @@ static inline void _rtc_set_enabled(bool on)
 #ifdef CPU_SAMD21
 static void _rtc_clock_setup(void)
 {
-    /* Use 1024 Hz GCLK4 */
-    GCLK->CLKCTRL.reg = GCLK_CLKCTRL_CLKEN | GCLK_CLKCTRL_GEN(4) | GCLK_CLKCTRL_ID_RTC;
+    /* Use 1024 Hz GCLK3 */
+    GCLK->CLKCTRL.reg = GCLK_CLKCTRL_CLKEN | GCLK_CLKCTRL_GEN(3) | GCLK_CLKCTRL_ID_RTC;
```

Comment:
Is this `SAM0_GCLK_1KHZ`? If yes then we should either add that in the comment, or change the 3 to `SAM0_GCLK_1KHZ`.